lib/gui/modules/analytics.js

```diff
@@ -141,21 +79,7 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    * });
    */
   this.logEvent = (message, data) => {
-    const debugData = utils.hideAbsolutePathsInObject(utils.makeFlatStartCaseObject(data));
```

Comment:
Hm, it looks like you even deleted `utils.hideAbsolutePathsInObject` here. Is this function completely unused now? Should we move it to resin raven?